actions/regex-validator/tests/validate-input.test.ts

```diff
@@ -58,8 +58,8 @@ describe('Validate Input Required True', () => {
 
 describe('Error Message', () => {
     it('displays error', () => {
-        const regexExpression = "^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$"
-        var result = ValidateInput(input, regexExpression, true);
+        const regexExpression = String.raw`!\\\\\\^!^!^(0|[1-9]\d*)$?`
+        var result = ValidateInput('cdbbdbsbz', regexExpression, true);
```

Comment:
Did we want to use the value defined on line 5 (`const input = 'cdbbdbsbz';`)?

Reply:
No, it's a global variable that probably should have been made local to each test.  I'll refactor that sometime in the future.